### DISCUSSION TITLE
Chapter: "Alexa Talking to Home Assistant (The Skill)" - Not working at all!

### DISCUSSION THREAD

**Discussion by Flutzi** [keatontaylor/alexa-actions#241] [Q&A]
Hello. 

I hope you can help me. During setup, I followed everything step by step. In the third chapter "Alexa Talking to Home Assistant (The Skill)", I worked through everything from start to finish exactly. 

Now, when I try to test the skill and I enter the command "öffne actionable notifications" (in Englisch: "open actionable notifications") in the "Alexa Development Console" under "Test", it responds with: "Bei der Antwort des angeforderten Skill ist ein Problem aufgetreten." (in English: "There was a problem with the response of the requested skill!"). 

I have provided both my server (set up via cloudflared) and the token as described. Nevertheless, this error message appears every time after about 2-3 seconds. 

Can someone tell me what's causing this? That would be really awesome. Thank you very much in advance for your help!

**Comment by DEADSEC-SECURITY** [comment]
I'm closing as I think this was fixed in an issue already.
